### ISSUE TITLE
Conceptual Documentation Questions

### ISSUE THREAD

**Issue by inrazhtet** [MDRCNY/pump_validate#59]
## Question

1) @portermdrc Kristin, you have already explained this to me but, could you remind/verify with me again why when we do permutation for H0 vs H1 for the multiple adjustments we don't really reassignment treatment variables.
Instead, what we have is H0 drawn from a multivariate distribution and shifted by the mean of H1. 
How is it equivalent to a permutation of treatment reassignment to outcomes? Sorry, I looked back at my notes and I forgot to write down what you told me but, I stared at it and make some guesses below.

### Code Block Below

`t.shift<-t.mean.H1(MDES,J,n.j,R2.1,p)`
`t.df<-df(J,n.j,numCovar.1)`
`t.shift.mat<-t(matrix(rep(t.shift,tnum),M,tnum))`
`Zs.H0<- mvtnorm::rmvt(tnum, sigma = sigma, df = t.df, delta = rep(0,M),type = c("shifted", "Kshirsagar"))`
`Zs.H1 <- Zs.H0 + t.shift.mat`
`pvals.H0<- stats::pt(-abs(Zs.H0),df=t.df) * 2`
`pvals.H1<- stats::pt(-abs(Zs.H1),df=t.df) * 2`
`abs.Zs.H0 <- abs(Zs.H0)`
`abs.Zs.H1 <- abs(Zs.H1)`

### My Guess

In the WestFall-Young case, we are comparing 1 row of raw test statistics under the alternative hypothesis to 10,000 permutation of test statistics under the null hypothesis. 

We only need to generate permutation of 10,000 test statistics of the null without explicit treatment reassignment because in essence, we are constructing a multivariate distribution. We then compared each instance of the multivariate distribution to see whether it has values as extreme as the raw test statistics under the alternative. We note**TRUE** boolean for those cases under the null and obtain a likelihood out of 10,000 permutation.
Then, we run this iteration on the number of samples we have (in our case, the default is 1,000 but for computation simplicity, I have it at 10). 

In short, the p-values (i.e likelihood against the 10,000 permutation) generated represent adjusted cut-off points for any M outcome hypothesis testing. 

In conclusion, we don't need to have treatment reassignment in our permutation because we are assuming the null to be true and we are trying to estimate the current likelihood of extreme values given a raw alternative when we have multiple outcomes.

I hope this sort of makes sense.

In any event, no worries and no rush on answering me back. I have both emotionally and mentally moved on from this for now. Each time, there is a little something more to think about.

Thanks!!

Zarni
